docs/general/adapters.md

```diff
@@ -30,6 +30,8 @@ resources in the `"included"` member when the resource names are included in the
   render @posts, include: 'authors,comments'
 ```
 
+The format of the `include` option can be either a String composed of a comma-separated list of [http://jsonapi.org/format/#fetching-includes](relationship paths), an Array of Symbols and Hashes, or a mix of both.
```

Comment:
@beauby  I think this link is wrong, isn't?

I believe the right way is something like that:
```
[relationship paths](http://jsonapi.org/format/#fetching-includes)
```